problems/5.loops.js

```diff
@@ -2,18 +2,14 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
-
-}
+function evenNumbers() {}
```

Comment:
I'm assuming you ran out of time for this part. Would love for you to review the loops lesson tackle these over break